apps/web/__e2e__/editor.test.ts

```diff
@@ -84,6 +84,87 @@ test("creating a new note should clear the editor contents & title", async ({
   expect(await notes.editor.getContent("text")).toBe("");
 });
 
+test("when autosave is disabled, pressing ctrl+s should save the note", async ({
+  page
+}) => {
+  const app = new AppModel(page);
+  await app.goto();
+  const notes = await app.goToNotes();
+  const content = "a ".repeat(100);
+  await notes.createNote({
+    title: NOTE.title,
+    content
+  });
+
+  await page.locator(getTestId("editor-save-state-notsaved")).waitFor();
+  await page.locator(getTestId("editor-save-state-saved")).waitFor({
+    state: "hidden"
+  });
+
+  page.keyboard.press("Control+s");
+
+  await page.locator(getTestId("editor-save-state-notsaved")).waitFor({
+    state: "hidden"
+  });
+  await page.locator(getTestId("editor-save-state-saved")).waitFor();
+});
+
+test("when autosave is disabled, switching to another note should save the note", async ({
```

Comment:
This test should go like this:

1. Open app
2. Create 2 notes
3. Switch to note 1 and write until autosave is disabled
4. Switch to note 2
5. See if note 1 gets saved or not